Jenkinsfile

```diff
@@ -2,7 +2,7 @@ buildMvn {
   publishModDescriptor = 'yes'
   publishAPI = 'no'
   mvnDeploy = 'yes'
-  runLintRamlCop = 'yes'
+  runLintRamlCop = 'no'
```

Comment:
The new api-lint (https://dev.folio.org/guides/api-lint/) is the replacement. Please add it when your src is ready.

Reply:
Hi @dcrossleyau , thank you for so helpful notes. I fixed all of them. Could you please take a final look again to ensure that I did everything correctly?) Thank you